### PR TITLE
Add openshift-kni quay.io image mirror mapping

### PR DESCRIPTION
**Pull request by davidvossel**

### CODE REVIEW

File: core-services/image-mirroring/openshift-kni/OWNERS [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```
Comment (hongkailiu) [inline core-services/image-mirroring/openshift-kni/OWNERS]: Maybe remove this line (not true for this folder)?

Comment (davidvossel) [reply core-services/image-mirroring/openshift-kni/OWNERS]: yep, fixed

**Comment by hongkailiu** [comment]
/lgtm

waiting for the tests ...

/hold

**Pull request merged**